sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/CopyOperationResult.Serialization.cs

```diff
@@ -19,7 +19,7 @@ internal static CopyOperationResult DeserializeCopyOperationResult(JsonElement e
             OperationStatus status = default;
             DateTimeOffset createdDateTime = default;
             DateTimeOffset lastUpdatedDateTime = default;
-            CopyResult copyResult = default;
+            Optional<CopyResult> copyResult = default;
```

Comment:
So this property is not a collection. Is it affected by the change because it is not marked as required in the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json#L758)?

Reply:
The deserialization code changed a bit for all non-required properties, but public API surface only changed for collections.